clean/acadia_so_cprr.py

```diff
@@ -0,0 +1,52 @@
+import sys
+sys.path.append('../')
+from lib.path import data_file_path
+import pandas as pd
+from lib.columns import clean_column_names, set_values
+from lib.clean import clean_dates
+from lib.uid import gen_uid
+
+
+def clean_names(df):
+    names = df.deputy.str.lower().str.strip().fillna('')\
+        .str.extract(r'(\w+) ?(\w+)? (\w+)')
+    
+    df.loc[:, 'first_name'] = names[0].fillna('')
+    df.loc[:, 'middle_name'] = names[1].fillna('')
+    df.loc[:, 'last_name'] = names[2].fillna('')
+    return df[~((df.first_name ==  '') & (df.last_name ==  ''))].drop(columns='deputy')
+
+
+def clean_allegation(df):
+    df.loc[:, 'allegation'] = df.allegation.str.lower().str.strip()\
+        .str.replace(r'\bia\b', 'internal affairs', regex=True)
+    return df 
+
+
+def clean_action(df):
+    df.loc[:, 'action'] = df.action.str.lower().str.strip()\
+        .str.replace(r'^(\w{1}) day', r'\1-day', regex=True)
+    return df 
+
+
+def clean():
+    df = pd.read_csv(data_file_path('raw/acadia_so/cprr_acadia_so_2018_2021.csv'))\
+        .pipe(clean_column_names)\
+        .rename(columns={
+            'date': 'receive_date'
+        })\
+        .pipe(clean_dates, ['receive_date'])\
+        .pipe(clean_names)\
+        .pipe(clean_allegation)\
+        .pipe(clean_action)\
+        .pipe(set_values, {
+            'agency': 'acadia parish so'
```

Comment:
Shouldn't the agency name be in title case? So maybe `Acadia Parish SO` instead?

Reply:
Yes. I should have held off on branch this until we've standardized agency names re: the POST branch. Thanks.